digid_eherkenning/management/commands/generate_digid_metadata.py

```diff
@@ -204,6 +204,7 @@ def generate_metadata(self, options):
             "security": {
                 "signMetadata": True,
                 "authnRequestsSigned": True,
+                "logoutRequestSigned": True,
```

Comment:
I don't see any changes in the metadata if we include this option here or not :thinking: 

I had a look again here: https://github.com/maykinmedia/python3-saml/blob/master/README.md#initiate-slo

I think this setting is used when we are making the request to sign out, to know whether we should sign the request or not. So I don't think this setting is needed in the metadata generation.